tldw_Server_API/app/core/Chat/chat_history.py

```
@@ -42,6 +50,11 @@ def save_chat_history_to_db_wrapper(
 ) -> Tuple[Optional[str], str]:
     """
     Persist a chat history into the ChaChaNotes database, creating or updating a conversation record.
+
+    This function is maintained for legacy integrations and Character Chat
+    utilities. For new features built on `/api/v1/chat/completions`, prefer
+    using the Chat module’s conversation/message persistence via
```

Ruff (0.14.5)

Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)